Повна стилізація футера

maridra opened the pull request.

maridra committed:
Повна стилізація футера

Стилізація футера десктопної, планшетної та мобільної версії

maridra committed:
Повна стилізація футера

maridra committed:
Додано bg img на футер десктопної версії

maridra committed:
Merge branch 'main' into img-md

jekiekolya reviewed (changes requested):
https://www.loom.com/share/d917dd819b7e4911a39bb2f81aa6fdcc

maridra committed:
Додані зміни за корективами у футеріі

Змінено позиціонування bg img в десктопної версії.
Додані падінги на посилання у навігації за сторінкою.
Інпут та кнопка у формі розтягується до ширини 480px у мобільній версії.
Висота інпута змінена до правильного значення за допомогою падінгу

jekiekolya reviewed (approved):
ок
